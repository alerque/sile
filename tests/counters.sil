\begin[papersize=a6]{document}
\nofolios
\set[parameter=document.parindent,value=0pt]
\font[family=Libertinus Serif]
\script[src=packages/counters]
\set-counter[id=test,value=1234]

Default: \show-counter[id=test]

Alpha: \show-counter[id=test,display=alpha]

N'th: \show-counter[id=test,display=nth]

Roman: \show-counter[id=test,display=ROMAN]

Greek: \font[language=el]{\show-counter[id=test,display=greklow]}

Japanese: \font[family=Noto Sans CJK JP,language=ja]{\show-counter[id=test,display=jpan]}

Arabic-Indic: \font[family=Noto Naskh Arabic,language=ar]{\show-counter[id=test,display=arabext]}

English String: \show-counter[id=test,display=String]

Turkish String: \font[language=tr]{\show-counter[id=test,display=String]}

Turkish Ordinal: \font[language=tr]{\show-counter[id=test,display=Ordinal]}

\end{document}
