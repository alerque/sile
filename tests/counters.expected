Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	27.9564
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	37 70 71 66 86 77 85	(Default)
Mx 	45.1419
T	27	(:)
Mx 	50.5286
T	18 19 20 21	(1234)
Mx 	14.8819
My 	39.9564
T	34 77 81 73 66	(Alpha)
Mx 	39.6119
T	27	(:)
Mx 	45.0000
T	66 86 77	(aul)
Mx 	14.8819
My 	51.9564
T	47 8 85 73	(N'th)
Mx 	32.3119
T	27	(:)
Mx 	37.6988
T	18 19 20 21	(1234)
Mx 	56.2988
T	1756	(’)
Mx 	58.9788
T	85 73	(th)
Mx 	14.8819
My 	63.9564
T	51 80 78 66 79	(Roman)
Mx 	43.6819
T	27	(:)
Mx 	49.0647
T	46 36 36 57 57 57 42 55	(MCCXXXIV)
Mx 	14.8819
My 	75.9564
T	40 83 70 70 76	(Greek)
Mx 	39.5019
T	27	(:)
Mx 	44.8883
T	811	(͵)
Mx 	46.8083
T	859 877 869 862	(ασλδ)
Mx 	68.3883
T	116	(´)
Mx 	14.8819
My 	87.9564
T	43 66 81 66 79 70 84 70	(Japanese)
Mx 	50.6919
T	27	(:)
Mx 	56.0727
Set font 	Noto Sans CJK JP;10;400;;normal;;LTR
T	11657	(千)
Mx 	66.0935
T	9669	(二)
Mx 	76.1144
T	27692	(百)
Mx 	86.1352
T	9492	(三)
Mx 	96.1560
T	11655	(十)
Mx 	106.1768
T	13131	(四)
Mx 	14.8819
My 	99.9564
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	34 83 66 67 74 68	(Arabic)
Mx 	42.0419
T	14	(-)
Mx 	45.4219
T	42 79 69 74 68	(Indic)
Mx 	65.8619
T	27	(:)
Mx 	71.2460
Set font 	Noto Naskh Arabic;10;400;;normal;;LTR
T	241	(۱)
Mx 	74.4101
T	108	(٬)
Mx 	76.0165
T	242 243 244	(۲۳۴)
Mx 	14.8819
My 	111.9564
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	38 79 72 77 74 84 73	(English)
Mx 	48.5059
T	52 85 83 74 79 72	(String)
Mx 	73.3659
T	27	(:)
Mx 	78.7298
T	48 79 70	(One)
Mx 	98.6438
T	53 73 80 86 84 66 79 69	(Thousand)
Mx 	142.2978
T	53 88 80	(Two)
Mx 	163.2118
T	41 86 79 69 83 70 69	(Hundred)
Mx 	202.5458
T	53 73 74 83 85 90	(Thirty)
Mx 	228.6358
T	14	(-)
Mx 	232.0158
T	39 80 86 83	(Four)
Mx 	14.8819
My 	123.9564
T	53 86 83 76 74 84 73	(Turkish)
Mx 	49.6159
T	52 85 83 74 79 72	(String)
Mx 	74.4759
T	27	(:)
Mx 	79.8499
T	35 74 79	(Bin)
Mx 	96.8739
T	240 76 74	(İki)
Mx 	110.6879
T	58 188 91	(Yüz)
Mx 	129.0019
T	48 85 86 91	(Otuz)
Mx 	151.7459
T	37 182 83 85	(Dört)
Mx 	14.8819
My 	135.9564
T	53 86 83 76 74 84 73	(Turkish)
Mx 	49.6124
T	48 83 69 74 79 66 77	(Ordinal)
Mx 	80.6724
T	27	(:)
Mx 	86.0428
T	35 74 79	(Bin)
Mx 	103.0633
T	240 76 74	(İki)
Mx 	116.8738
T	58 188 91	(Yüz)
Mx 	135.1842
T	48 85 86 91	(Otuz)
Mx 	157.9247
T	37 182 83 69 188 79 68 188	(Dördüncü)
End page
Finish
